plugins/provisioners/salt/default_downloader.rb

```diff
@@ -0,0 +1,93 @@
+require 'open-uri'
+require 'digest'
+require_relative "./errors"
+
+module VagrantPlugins
+  module Salt
+    class DefaultDownloader
```

Comment:
Taking name suggestions

Reply:
`BootstrapDownloader` seems alright since this is specifically for that.